server/src/utils/helper.ts

```diff
@@ -0,0 +1,3 @@
+export function getParagraphText(paragraphIdx?: number): string {
+  return `${paragraphIdx ? '' : 'next'} paragraph ${paragraphIdx ? paragraphIdx : ''} `
+}
```

Comment:
When working again on this, let's add a descriptive comment on what it does (as it took me a while) and possibly change its name to something such as getParagraphIdentifier